css/style.css

```diff
@@ -94,33 +171,47 @@ h2 {
 .down-arrow img {
   border: 1px solid #dfe1e6;
   border-radius: 32px;
-}
-
-.work-container {
-  display: grid;
+  width: 40px;
+  height: 40px;
 }
 
 .work-title {
   display: flex;
   justify-content: center;
+  padding: 0 24px;
 }
 
 .work-title h2 {
   text-align: center;
-  padding: 0 24px;
 }
 
 .work-list {
-  display: flex;
+  display: grid;
+  grid-template-columns: auto;
+  padding: 24px 22px;
 }
 
-.work-list ul {
-  list-style-type: none;
-  margin: 0;
-  padding: 0;
+/** display grid for medium version **/
+@media only screen and (min-width: 768px) and (max-width: 1200px) {
+  .work-list {
+    grid-template-columns: repeat(2, 369px);
+    justify-content: space-between;
+    column-gap: 24px;
+    row-gap: 32px;
+  }
 }
 
-.work-list .list-item {
+/** display grid for large desktop devices **/
+@media only screen and (min-width: 1200px) {
+  .work-list {
+    grid-template-columns: repeat(3, 369px);
+    justify-content: space-between;
+    column-gap: 24px;
+    row-gap: 32px;
+  }
+}
+
+.list-item {
```

Comment:
Great implementation and usage of breakpoints in CSS 👏🏻 
- [ ] Let's kindly make sure our cards have a space at the bottom to match Figma design

Your implementation | Figma Design
:------:|:-----:
![image](https://user-images.githubusercontent.com/87689547/160120551-438c09c3-cff7-48f6-89c3-b31d8a43cfd9.png) | ![image](https://user-images.githubusercontent.com/87689547/160120703-d6d25eb1-c747-47b5-b901-27e31bf1147f.png)